command/agent/profile/pprof.go

```diff
@@ -0,0 +1,133 @@
+package profile
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"runtime/pprof"
+	"runtime/trace"
+	"strings"
+	"time"
+)
+
+// goroutine
+// threadcreate
+// heap
+// allocs
+// block
+// mutex
```

Comment:
Will remove this altogether, this was leftover scratch pad. They aren't consts because they are just valid examples of profile lookup requests. func Profile has a comment to see `runtime/pprof/pprof.go` for available profiles